k8s/django-config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: django-config
+data:
+  AUTO_REINDEX_INTERVAL: "5400"
+  CELERY_BROKER_URL: "redis://localhost:6379"
+  CELERY_RESULT_BACKEND: "redis://localhost:6379"
+  CONTACT_EMAIL: "tools-help@ietf.org"
+  DATASET_TMP_ROOT: "/data/datasets"
+  DATATRACKER_CLIENT_ID: ""
+  DEBUG: "0"
+  INTERNAL_HOSTNAMES: "localhost,bib.bib.svc.cluster.local,127.0.0.1"
+  MATOMO_SITE_ID: ""
+  MATOMO_TAG_MANAGER_CONTAINER: ""
+  MATOMO_URL: "analytics.ietf.org"
+  PORT: "8000"
+  PRIMARY_HOSTNAME: "bib.ietf.org"
+  PYTHONUNBUFFERED: "1"
+  REDIS_HOST: "localhost"
+  REDIS_PORT: "6379"
+  SENTRY_DSN: ""
+  SERVER_EMAIL: "support@ietf.org"
+  SERVICE_NAME: "IETF BibXML Service"
+  SOURCE_REPO_URL: "https://github.com/ietf-tools/bibxml-service"
+  SNAPSHOT: "1.3.0"
```

Comment:
Can we use `APP_IMAGE_TAG` for this?

Reply:
Config map has been deprecated in favour of the external secret, hence resolved.